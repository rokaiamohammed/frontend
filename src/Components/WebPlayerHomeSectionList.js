import React, {Component} from 'react';
import Section from '../Components/WebPlayerHomeSection';
// import '../Components/WebPlayerHome.css';
import {useDispatch,useSelector} from 'react-redux'

const WebPlayerHomeSectionList = ({SectionTypes}) =>{
    // Dispatch(LoadSongs({id:'1',url:'1',name:'1',artist:'1'}))

    return (
        <div>
            {
                SectionTypes.map((user,i) =>{
                    return(
                        <Section 
                            key={i}
                            // id={SectionTypes[i].id}
                            title={SectionTypes[i].title}
                            // des={SectionTypes[i].des}
                            type={SectionTypes[i].data}
                        />
                    );
                })
            }
        </div>
    )
}
export default  WebPlayerHomeSectionList;