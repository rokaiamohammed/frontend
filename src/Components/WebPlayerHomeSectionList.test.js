import React from 'react';
import { shallow, mount } from 'enzyme';
import WebPlayerHomeSectionList from './WebPlayerHomeSectionList';

const SectionTypes = [{
    key: 1,
    title: 'Top hits',
    type: 'no description',

  }]

it('WebPlayerHomeSectionList renders without crashing', () => {
  expect(shallow(<WebPlayerHomeSectionList SectionTypes={SectionTypes} />)).toMatchSnapshot();
});