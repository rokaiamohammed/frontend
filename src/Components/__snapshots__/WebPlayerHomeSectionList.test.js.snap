// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebPlayerHomeSectionList renders without crashing 1`] = `ShallowWrapper {}`;

exports[`renders without crashing 1`] = `ShallowWrapper {}`;
